File: 250907/짝수인 것만 출력하기/print-even-number.py
n=int(input())
arr=list(map(int, input().split()))
new_arr=[]
for i in range(n):
    if arr[i]%2==0:
        new_arr.append(arr[i])
for i in range(len(new_arr)):
    print(new_arr[i],end=" ")

File: 260416/짝수인 것만 출력하기/print-even-number.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin>>n;
    int arr[n];
    int newarr[n];
    int a=0;
    for(int i=0;i<n;i++){
        cin>>arr[i];
        if (arr[i]%2==0){
            newarr[a]=arr[i];
            a++;
        }
    }

    for(int i=0;i<a;i++){
        cout<<newarr[i]<<" ";
    }
    // Please write your code here.
    return 0;
}